caddy/https/https.go

```diff
@@ -401,21 +401,21 @@ var (
 // default port for the challenge must be forwarded to this one.
 const AlternatePort = "5033"
 
-// KeySize represents the length of a key in bits.
-type KeySize int
-
-// Key sizes are used to determine the strength of a key.
+// Possible types for a key.
 const (
-	Ecc224  KeySize = 224
-	Ecc256          = 256
-	Rsa2048         = 2048
-	Rsa4096         = 4096
+	RSA2048		= acme.RSA2048
+	RSA4096		= acme.RSA4096
+	EC256		= acme.EC256
+	EC384		= acme.EC384
```

Comment:
We might not need these. Why not just use the values from the `acme` package directly?